integration/integration_test.go

```diff
@@ -1553,6 +1553,7 @@ func testIPPropagation(t *testing.T, suite *integrationTestSuite) {
 		tc.Stdout = person
 		tc.Stdin = person
 
+		//nolint:staticcheck // SA1019. There is here to test the SSH connection path as long as it exists.
```

Comment:
```suggestion
		//nolint:staticcheck // SA1019. It's here to test the SSH connection path as long as it exists.
```

I think? Same for others.